src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1139 @@
+
+    There is no feature in OpenTelemetry that is analogous to New Relic’s APM SQL Traces. The SQL Traces UI is where explain plans would be displayed if they were captured, so this feature is also not supported for OpenTelemetry.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="explains-plans"
+    title="Explain plans"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+    (See above)
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="obfuscation"
+    title="Obfuscation"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+    Currently, there is not a uniform way in OTel to obfuscate sensitive information. Check each instrumented library you are using to see if there is a configuration option for query sanitization. 
+  </Collapser>
+</CollapserGroup>
+
+## Externals [#externals]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+The external services feature allows you to look closely at the upstream and downstream activity of a single service. Whether you’re monitoring with New Relic agents or OpenTelemetry, the external services feature shows transaction-level detail as you drill into connected services.
+
+For OpenTelemetry, this view is based strictly on span data (which may be sampled), while New Relic agents report unsampled metrics on the interservice connections.
+
+## Logs [#logs]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+## Service map [#service-maps]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+This map displays an entire set of interconnected services for a given service. The interconnected services must be actively reporting to appear in this map. Compare with these features: related entities and automap.
+
+## Dependencies [#dependencies]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+For OpenTelemetry, the **Related entities** view can take you to all the dependencies for a given service.
+
+## Errors [#errors]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+For OpenTelemetry services in the **Errors** page, the error rate is based off span data. Traces will be displayed if they contain a span or spans that meet the following conditions:
+
+* `otel.status_code=ERROR`
+* `span.kind=server` OR `span.kind=consumer`
+
+The above also applies to exceptions, which are displayed as a span event on the span it was recorded on.
+
+If you are not seeing an error that you expect to see in the **Errors** page, check the distributed tracing page since it does not exclude based on the `span.kind` attribute.
+
+
+## Errors inbox [#errors-inbox]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+## Metrics explorer [#metrics-explorer]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
```

Comment:
I think the `Metrics explorer` section stands alone and has just the one entry, about timeslice vs. dimensional. `Metrics with transaction/endpoint dimensions` and `Metrics on aggregate performance of spans (“transaction segments”)` should live under a separate `Metrics for services` section, IMO.